tests/integration/test.py

```diff
@@ -56,7 +61,6 @@ def test_01_help_list(self):
                 self.assertEqual(r, None)
 
 
-class TestTemplate(unittest.TestCase):
```

Comment:
It was a mistake as a wrong indentation line 54 (image.set_globals(api, login, password))